titan-options.php

```diff
@@ -489,44 +495,116 @@ function regala_create_options() {
 	/**
 	 *   Sections
 	 */
-    
-    $havanaSections = $titan->createThemeCustomizerSection( array(
-        'name' => __( 'Sections', 'regala' ),
-        'panel' => __( 'Theme Options & Colors', 'regala' ),
-    ) );
-    
-    $sections = $titan->createThemeCustomizerSection( array(
-        'name' => __( 'Section 1', 'regala' ),
-        'panel' => __( 'Sections', 'regala' ),
-    ) );
-    
-    $sections->createOption( array(
-        'name' => __( 'Layout', 'regala' ),
-        'id' => 'section_1',
-        'type' => 'radio-image',
-        'options' => array(
-            'layout1' => get_template_directory_uri() . '/images/header-image-default1-thumb.jpg',
-            'layout2' => get_template_directory_uri() . '/images/header-image-default1.jpg',
-            'layout3' => get_template_directory_uri() . '/images/header-image-default2-thumb.jpg',
-            'layout4' => get_template_directory_uri() . '/images/header-image-default2.jpg',
-            'layout5' => get_template_directory_uri() . '/images/logo.png',
-            'layout6' => get_template_directory_uri() . '/images/site-icon.png',
-         ),
-    ) );    
-  
-    
-	
+         
+    for ( $sectionIndex = 1; $sectionIndex <= 5; $sectionIndex++ ) {
+        $sectionLayout = $titan->createThemeCustomizerSection( array(
+             'name' => sprintf( __( 'Section %d Layout ', 'havana' ), $sectionIndex ),
+             'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+          ) );
+          
+          $sectionLayout->createOption( array(
+              'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+              'id' => 'section' . $sectionIndex . 'layout',
+              'type' => 'radio-image',
+              'options' => array(
+                  'layout1' => get_template_directory_uri() . '/images/header-image-default1-thumb.jpg',
+              ),
+          ) );
+        
+        for ( $boxIndex = 1; $boxIndex <= 4; $boxIndex++ ) {
+                           
+             $sectionBox = $titan->createThemeCustomizerSection( array(
+                 'name' => sprintf(__( 'Section %1$s - Box %2$s Content', 'havana' ), $sectionIndex, $boxIndex ),
+                 'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+              ) );
+              
+              $sectionBox->createOption( array(
+                  'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+                  'id' => 'section' . $sectionIndex . 'box' . $boxIndex . 'post/page',
```

Comment:
Might be better also for readability if all the `id`s have underscores to separate the words in the `id`

e.g.
```'section_' . $sectionIndex . '_box_' . $boxIndex . '_type'